Alpha1202/pastebin-code-highlighting-django-rest-API-app
# create a method for the api root

#### What does this PR do?

- Creating an endpoint for the API root

#### Description of Task to be completed?

- create a method for the API root

#### How should this be manually tested?

- after cloning the repo, cd into it
- and run `python manage.py runserver`
- then navigate to `http://localhost:8000/` to see the API root view

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- N/A

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
